Python/compile.c

```diff
@@ -1545,6 +1564,11 @@ compiler_addop_j(struct compiler *c, int opcode, basicblock *b)
     } \
 }
 
+#define CHECK(X)  \
+    if (!(X)) {   \
+        return 0; \
+    }
```

Comment:
This comes in handy to keep visual clutter down in the nastier compound patterns.

Reply:
Could you give this a more meaningful name, please. E.g.  `RETURN_IF_FALSE`